### ISSUE TITLE
[Bug]: unknown error:[58000]'[1997-01-01 

### ISSUE THREAD

**Issue by daviszhen** [matrixorigin/matrixone#2197]
### Is there an existing issue for the same bug?

- [X] I have checked the existing issues.

### Environment

```markdown
- Version or commit-id (e.g. v0.1.0 or 8b23a93):ebb2eca68e8712efe2dff34f26fcc025ed5dcf01
- Hardware parameters:mac
- OS type:
- Others:
```

### Actual Behavior

strange errors:

mysql> SELECT sum(LO_REVENUE),year(LO_ORDERDATE) AS year,P_BRAND,count(*) FROM lineorder_flat WHERE P_CATEGORY = 'MFGR#12' AND S_REGION = 'AMERICA' GROUP BY year(LO_ORDERDATE),P_BRAND ORDER BY year,P_BRAND;
ERROR 1105 (HY000): unknown error:[58000]'[1997-01-01 1997-01-01 1997-01-01 1997-01-01 1997-01-01 1997-01-01 1997-01-01 1997-01-01 1997-01-01 1997-01-01 1997-01-01 1997-01-01 1997-01-01 1997-01-01 1997-01-01 1997-01-01 1997-01-01 1997-01-01 1997-01-01 1997-01-01 1997-01-01 1997-

01-01 1997-01-01 1997-01-01 1997-01-01 1997-01-01 1997-01-01 1997-01-01 1997-01-01 1997-01-01 1997-01-01 1997-01-01 1997-01-01 1993-01-01 1993-01-01 1993-01-01 1993-01-01 1993-01-01 1993-01-01 1993-01-01 1993-01-01 1993-01-01 1993-01-01 1993-01-01 1993

### Expected Behavior

select can be executed finely.

### Steps to Reproduce

```markdown
three node with aoe.
```

### Additional information

data file:
[lineorder_flat_1w.csv.zip](https://github.com/matrixorigin/matrixone/files/8477974/lineorder_flat_1w.csv.zip)

load script:
[load-tpe1w.sql.zip](https://github.com/matrixorigin/matrixone/files/8477976/load-tpe1w.sql.zip)

logs :
[log0.txt](https://github.com/matrixorigin/matrixone/files/8469894/log0.txt)
[log1.txt](https://github.com/matrixorigin/matrixone/files/8469896/log1.txt)
[log2.txt](https://github.com/matrixorigin/matrixone/files/8469898/log2.txt)

configs:
[conf2.txt](https://github.com/matrixorigin/matrixone/files/8469911/conf2.txt)
[conf0.txt](https://github.com/matrixorigin/matrixone/files/8469906/conf0.txt)
[conf1.txt](https://github.com/matrixorigin/matrixone/files/8469910/conf1.txt)

![1649742722732](https://user-images.githubusercontent.com/60595215/162890139-55a74a2c-d591-4141-9782-525feabfd84d.jpg)

**Comment by daviszhen** [comment]
It disappeared at commit `18d59acb9ffd8e2737bf987fe6b0e3b4ef4e5090`.

**Comment by qingxinhome** [comment]
After adding the serialization of the min's DateRing/DatetimeRing struct, the problem has been solved,For details, refer to commit:e1b31f750acb5c906789d77ec4dda74ffb41c63c
